loader/loader.c

```diff
@@ -2910,8 +2910,10 @@ static VkResult read_data_files_in_search_paths(const struct loader_instance *in
     bool xdg_data_dirs_secenv_alloc = true;
 #endif
 
-#ifdef _WIN32
+#if defined(_WIN32)
+#if WINVER >= _WIN32_WINNT_WINBLUE
```

Comment:
This is more involved than what I would've done. I would've kept the logic in `loader.c` and `loader_windows.h` the same, and just stubbed out `windows_get_app_package_manifest_path` to return `NULL` on builds that use the Win7 SDK.

Reply:
I did this cause I didn't want to take any chances. Since the win7 build is external, I don't have a good way of checking if my solution works.